dkg/sync/server.go

```diff
@@ -124,6 +137,28 @@ func NewServer(ctx context.Context, tcpNode host.Host, peers []p2p.Peer, defHash
 
 			log.Debug(ctx, "Message received from client", z.Any("client", p2p.PeerName(pID)))
 
+			if msg.Shutdown {
+				resp := &pb.MsgSyncResponse{
+					SyncTimestamp: msg.Timestamp,
+					Error:         "",
+				}
+
+				resBytes, err := proto.Marshal(resp)
+				if err != nil {
+					log.Error(ctx, "Marshal server response", err)
+					return
+				}
+
+				_, err = s.Write(resBytes)
+				if err != nil {
+					log.Error(ctx, "Send response to client", err, z.Any("client", p2p.PeerName(pID)))
+					return
```

Comment:
discussion: why not return the `error` as well?

Reply:
stream handler function doesn't return error, that's why only log is there